pandas/plotting/_core.py

```diff
@@ -1582,7 +1602,7 @@ def pie(self, **kwargs):
             raise ValueError("pie requires either y column or 'subplots=True'")
         return self(kind="pie", **kwargs)
 
-    def scatter(self, x, y, s=None, c=None, **kwargs):
+    def scatter(self, x, y, **kwargs):
```

Comment:
```suggestion
    def scatter(self, x, y, s=None, c=None, **kwargs):
```
need to keep the original position arguments to maintain backwards compat for those using positional args.